test/e2e/conftest.py

```diff
@@ -196,6 +196,7 @@ def worker_config(
             service_model_path=dst_path,
             file_mappings=file_mappings or None,
             windows_job_users=windows_job_users,
+            start_service=True,
```

Comment:
The PR description indicates that you only ran the cross-os tests, but this change will impact the os-specific tests as well. Can you please run those and verify that the change does not impact them negatively?

Reply:
Just ran them, they passed so it did not impact them